.flake8

```diff
@@ -60,13 +60,15 @@ per-file-ignores =
   # WPS412 logic of an extension is in __init__.py file;
   # FIXME: WPS201 too many imports
   # FIXME: WPS402 too many `noqa`s
-  src/sphinxcontrib/towncrier/__init__.py: E800, WPS201, WPS402, WPS412
+  # FIXME: WPS202 too many module members
+  src/sphinxcontrib/towncrier/__init__.py: E800, WPS201, WPS402, WPS412, WPS202
 
   # There are multiple `assert`s (S101)
   # and subprocesses (import – S404; call – S603) in tests;
   # also, using fixtures looks like shadowing the outer scope (WPS442);
+  # furthermore, we should be able to import and test private attributes (WPS450)
```

Comment:
For now, don't add this. Instead, ignore inline in a specific place. Adding global-ish ignores should be the last resort or should have a really good reason.

Besides, if you just follow the example of the `_towncrier` module, maybe there would be no need in ignoring anything at all.

Reply:
I gave it some thought, and maybe it's reasonable to have this ignore just for the tests directory. I'll cherry-pick it separately, though. Because it doesn't seem to be something directly related to the PR purpose. It's better if it remains atomic.